audio/audio_hw.c

```diff
@@ -396,9 +398,84 @@ static int out_set_parameters(struct audio_stream *stream, const char *kvpairs)
     return 0;
 }
 
-static char * out_get_parameters(const struct audio_stream *stream, const char *keys)
+static void str_parms_add_audio_format(struct str_parms* str_parms, 
```

Comment:
Rename. The prefix str_parms_*() is used for Android library functions so shouldn't overload that prefix with custom functions.

Reply:
Done.